src/main/java/org/mockito/verification/JUnitVerificationCollector.java

```diff
@@ -0,0 +1,108 @@
+package org.mockito.verification;
```

Comment:
Since it is JUnit specific, this whole PR should live within the package `org.mockito.junit`, we'd like to move runners as well under this package too.